src/features/jserrors/aggregate/index.js

```diff
@@ -197,98 +197,94 @@ export class Aggregate extends AggregateBase {
     var type = internal ? 'ierr' : 'err'
     var newMetrics = { time }
 
-    // sr, stn and spa aggregators listen to this event - stn sends the error in its payload,
-    // and spa annotates the error with interaction info
-    const msg = [type, bucketHash, params, newMetrics]
-    handle('errorAgg', msg, undefined, FEATURE_NAMES.sessionTrace, this.ee)
-    handle('errorAgg', msg, undefined, FEATURE_NAMES.spa, this.ee)
-    handle('errorAgg', msg, undefined, FEATURE_NAMES.sessionReplay, this.ee)
-
+    // Trace sends the error in its payload, and both trace & replay simply listens for any error to occur.
+    const jsErrorEvent = [type, bucketHash, params, newMetrics, customAttributes]
+    handle('errorAgg', jsErrorEvent, undefined, FEATURE_NAMES.sessionTrace, this.ee)
+    handle('errorAgg', jsErrorEvent, undefined, FEATURE_NAMES.sessionReplay, this.ee)
     // still send EE events for other features such as above, but stop this one from aggregating internal data
     if (this.blocked) return
```

Comment:
Do we need to worry about spa (old or new) still having access to the error event if the error feature is blocked?  This change adds new behavior that stops the jserror event from flowing on if blocked